### DISCUSSION TITLE
Resume Bundle X ATS Scoring

### DISCUSSION THREAD

**Discussion by kanakmaheshwari3115** [recodehive/resume-pitch#97] [Q&A]
Name: Kanak Maheshwari
LinkedIn Link: https://www.linkedin.com/in/kanak-maheshwari-6b7922285/
Resume: 
[RESUME Kanak Maheshwari.docx](https://github.com/user-attachments/files/17016332/RESUME.Kanak.Maheshwari.docx)
GitHub Link: https://github.com/kanakmaheshwari3115
I am Second year student pursuing B.Tech. (CSE) at DTU (Delhi Technological University) looking for internships in software development engineer role.

**Comment by sanjay-kv** [comment]
[ATS Version of CV (26).docx](https://github.com/user-attachments/files/17089452/ATS.Version.of.CV.26.docx)
[kanak.pdf](https://github.com/user-attachments/files/17089453/kanak.pdf)

@kanakmaheshwari3115
